fingertip_pressure/scripts/rectangle_viz.py

```diff
@@ -1,4 +1,4 @@
-#! /usr/bin/python
```

Comment:
https://github.com/PR2/linux_networking/pull/5#discussion_r584112906 for all of these.

Reply:
yeah, I figured this one out, it turns out I had python-is-python2 installed, which messes with the python stuff. Installing python3-is-python under ubuntu fixes this issue, and the shebang works as intended, now.